src/applications/model/gaming-streaming-server.cc

```diff
@@ -128,6 +102,60 @@ GamingStreamingServer::EraseStatistics (void)
   m_totSentPackets = 0;
 }
 
+void
+GamingStreamingServer::AddNewTraffic (Ptr<RandomVariableStream> packetSize,
+                                      Ptr<RandomVariableStream> interArrivalTime)
+{
+  NS_LOG_FUNCTION (this);
+  m_pksList.push_back (packetSize);
+  m_iatList.push_back (interArrivalTime);
+}
+
+void
+GamingStreamingServer::Send (Ptr<RandomVariableStream> packetSize,
+                             Ptr<RandomVariableStream> interArrivalTime)
+{
+  NS_LOG_FUNCTION (this);
+
+  uint32_t pktSize = packetSize->GetInteger ()+12; //TODO: think about 12
+
+  SeqTsHeader header;
+  header.SetSeq (m_seq++);
+  NS_ABORT_IF (pktSize < header.GetSerializedSize ());
+  Ptr<Packet> packet = Create<Packet> (pktSize - header.GetSerializedSize ());
+  packet->AddHeader (header);
+  NS_ABORT_IF (packet->GetSize () != pktSize);
+
+  m_socket->Send (packet);
+  m_totSentPackets++;
```

Comment:
how does it differ from `m_seq`?

Reply:
This attribute saves the total number of sent packets by now.
But we don't use it. maybe it's better to remove it from the code.

Reply:
i don't understand what you mean. both of them are increased by every time the `Send` method is called, so they really hold the same info. why keeping them both, then?